README.md

```diff
@@ -1 +1,23 @@
-# Algorithm-Visualization
\ No newline at end of file
+# Algorithm-Visualization - AlgViz
+
+Make SVGs of data structures from your code.
+
+## Development
+
+This package is supported by [pipenv](https://github.com/pypa/pipenv) for development.
+
+#### Setup
+
+```
+pipenv install
+pipenv shell
+```
+
+#### Linting
+We rely on pipenv's built in Flake 8 code style checker.
+```
+pipenv check --style ./path/to/files/
+```
+
+## Use as a package
+We do not have a release right now, so to experiment you can add it via a git clone
```

Comment:
To install without using `pipenv`, end-users should run `python setup.py install`.
Developers should instead run `python setup.py develop` so that we don't need to reinstall.

Reply:
Thanks!